src/backend/distributed/transaction/worker_transaction.c

```diff
@@ -142,13 +155,37 @@ SendCommandListToAllWorkers(List *commandList)
 	foreach(workerNodeCell, workerNodeList)
 	{
 		WorkerNode *workerNode = (WorkerNode *) lfirst(workerNodeCell);
-		SendCommandListToWorkerInSingleTransaction(workerNode->workerName,
-												   workerNode->workerPort, extensionOwner,
-												   commandList);
+		PG_TRY();
+		{
+			SendCommandListToWorkerInSingleTransaction(workerNode->workerName,
```

Comment:
note that there's already a `SendOptionalCommandListToWorkerInTransaction`

Reply:
oh my bad, didnt know that. I will use it.

Reply:
I thought you wanted me to create such a method :)